packages/appstore/src/components/wallet-cards-carousel/wallet-cards-carousel.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useRef } from 'react';
+import { ProgressBarOnboarding, Icon, Text, WalletCard } from '@deriv/components';
+import { getWalletHeaderButtons } from 'Constants/utils';
+import { TWalletAccount } from 'Types';
+import { formatMoney } from '@deriv/shared';
+import Slider from 'react-slick';
+import './wallet-cards-carousel.scss';
+
+interface WalletCardsCarouselProps {
+    readonly items: TWalletAccount[];
+}
+
+export const WalletCardsCarousel = ({ items }: WalletCardsCarouselProps) => {
+    const [activePage, setActivePage] = React.useState(0);
+    const ref = useRef<Slider | null>(null);
+
+    const wallet_btns = getWalletHeaderButtons(items[activePage]?.is_virtual);
+
+    const sliderJSX = React.useMemo(() => {
```

Comment:
Since this is its own component, you can split this outside of `WalletCardsCarousel`, then you would no longer need `useMemo` to avoid recreating it on each render.

Reply:
@yashim-deriv created new component `CardsSlider`